operator/pkg/apis/controlplane/v1alpha1/controlplane.go

```diff
@@ -56,6 +56,7 @@ type Etcd struct {
 // custom flags for components running on master nodes like apiserver, KCM and
 // scheduler.
 type MasterSpec struct {
+	KMSKeyARN         *string    `json:"kmskey,omitempty"`
```

Comment:
Should we use keyname instead of arn? Will work across users/accounts.

Reply:
This is the `arn` we need to pass to aws-encryption-provider which gets supplied [here](https://github.com/kubernetes-sigs/aws-encryption-provider/blob/dee8cbcb643284037574f32181ed320e5ced0276/vendor/github.com/aws/aws-sdk-go/service/kms/api.go#L9823), is the keyname same as alias?

Reply:
>	// To specify a CMK, use its key ID, Amazon Resource Name (ARN), alias name,
	// or alias ARN. When using an alias name, prefix it with "alias/". To specify
	// a CMK in a different AWS account, you must use the key ARN or alias ARN.

Looks like all of them work. Perhaps we can call this `KeyId` for parity?